container_api/test-bridge-caller/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+#[macro_use]
+extern crate hdk;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+use hdk::holochain_core_types::json::JsonString;
+
+
+fn handle_call_bridge() -> JsonString {
+    hdk::call("test-callee", "greeter", "public", "token", "hello", JsonString::from("{}")).unwrap()
```

Comment:
"test-callee" is the handle set in the container config for this bridge. This actually has to be defined in the DNA but currently the container does not enforce these handles to match up (next PR). Then we will also have a way of defining the needed bridges in the define_zome! macro below. A bridge handle would then be like the declaration of binding: if declared in the zome as required, zome code can expect a call to such a bridge to work because the container would not load a config in which a matching bridge definition was missing..

Reply:
At this point, it kind of looks like hdk::call should accept a struct maybe, instead of so many params

Reply:
yep, though it will lose at least one again soon when the capability stuff is completely switched to tokens..

Reply:
exactly.  also the capability name is in the current call and that will go away too.